docs/static_site/src/pages/get_started/osx_setup.md

```diff
@@ -110,13 +110,10 @@ the guide in [Math Library Selection](build_from_source#math-library-selection).
 ```bash
 rm -rf build
 mkdir -p build && cd build
-cmake -GNinja -C ../config.cmake ..
-cmake --build . --parallel 8
+cmake -C ../config.cmake ..
+cmake --build .
```

Comment:
Is --build required?

Reply:
Yes. Alternatively you could call `ninja`. Ninja won't work on Windows though, whereas
`cmake --build .` is the same on all configurations.

Reply:
If that's the case can we just use the command `ninja` in this doc, since it's for linux-like system. One less option is always better for usability.